cft/misc/s3-bucket/test.tf

```diff
@@ -1,13 +1,23 @@
 resource "aws_s3_bucket" "mybucket" {
   bucket = "mybucket"
-  acl = "public"
+  acl    = "private"
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure there are no world-writeable AWS S3 Buckets for 'All Users' predefined Group**
 
**Vulnerability**
Misconfigured S3 buckets can leak private information to the entire internet or allow unauthorized data tampering / deletion.
 
**Remediation**
The Amazon S3 Block Public Access feature provides settings for access points, buckets, and accounts to help manage public access to Amazon S3 resources. By default, new buckets, access points, and objects don't allow public access. However, users can modify bucket policies, access point policies, or object permissions to allow public access. S3 Block Public Access settings override these policies and permissions to limit public access to these resources.

source: tenablecs-